File: android2_lesson03/src/java/java/com/example/ama/android2_lesson03/ui/markers/base/MarkerView.java
package com.example.ama.android2_lesson03.ui.markers.base;

import com.example.ama.android2_lesson03.repo.data.model.SimpleMarker;
import com.example.ama.android2_lesson03.ui.base.PocketMapView;

import java.util.ArrayList;

public interface MarkerView extends PocketMapView {
    void showMarkerList(ArrayList<SimpleMarker> markers);

    void showEditDialog(String dialogTitle, String dialogMessage, SimpleMarker marker);
}

File: android2_lesson03/src/kotlin/java/com/example/ama/android2_lesson03/ui/markers/base/MarkerView.kt
package com.example.ama.android2_lesson03.ui.markers.base

import com.example.ama.android2_lesson03.repo.data.model.SimpleMarker
import com.example.ama.android2_lesson03.ui.base.PocketMapView

interface MarkerView : PocketMapView {
    fun showMarkerList(markers: ArrayList<SimpleMarker>)
    fun showEditDialog(dialogTitle: String, dialogMessage: String, marker: SimpleMarker)
}